test/Project.toml

```diff
@@ -11,7 +11,7 @@ Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 [compat]
 Aqua = "0.8"
 MeshIntegrals = "0.13"
-Meshes = "0.50, 0.51"
+Meshes = "0.51"
```

Comment:
Can you comment on why Meshes.jl v0.51 is necessary now? If everything works with Meshes.jl v0.50 I would still like to support it.

Reply:
It’s not necessary. In the next major version I’d planned to pull the minimum CRS and Julia versions inline with those in Meshes, but requiring a newer Meshes isn’t actually necessary.

Reply:
In that case, I would just keep it like it is.

Reply:
We can also think about another GitHub Action, which tests if all compat bounds are still valid using https://github.com/julia-actions/julia-downgrade-compat. This action downgrades all dependencies to the minimal allowed versions and runs the tests.

Reply:
Versions reverted.

Reply:
I'm still seeing `Meshes = "0.51"` :thinking:

Reply:
Weird, I could have sworn that I commit that edit, but I’m not seeing it either. I’ll double check when I get on.

Reply:
Should be fixed now.

Reply:
Thanks!